### ISSUE TITLE
1.18.2 Overlord mod incompatible with required versions fireplacelib

### ISSUE THREAD

**Issue by DarkSagaz** [The-Fireplace-Minecraft-Mods/Fireplace-Lib#52]
### Mod Version

all latest versions

### What mod loader(s) are you seeing the problem on?

Fabric

### Mod Loader Version

Fabric - 0.14.8

### Problem Description

Overlord mod does not recognize fireplacelib as the correct version despite fireplacelib being the correct version

### Steps To Reproduce

1. Create new custom modpack profile in the curseforge launcher
2. Add Overlord-3.1.7+1.18.2.jar and FirePlaceLib into the pack (both FPL version fabric-7.0.1 and FPL version fabric-7.0.0 produce the same error)
3. run the profile
4. receive incompatible mod set error message

### Crash Report or Error Logs

```shell
Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'Fireplace Lib' (fireplacelib) 7.0.0+1.18.2 with any version between 6.2 (inclusive) and 7 (exclusive).
Unmet dependency listing:
	 - Mod 'Overlord' (overlord) 3.1.7+1.18.2 requires any version between 6.2 (inclusive) and 7 (exclusive) of mod 'Fireplace Lib' (fireplacelib), but only the wrong version is present: 7.0.0+1.18.2!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

**Comment by The-Fireplace** [comment]
Use Fireplace Lib 6 for now. I'm still working on the update.

**Comment by The-Fireplace** [comment]
Fixed with Overlord 3.2
